modules/ROOT/pages/secure-configuration-properties.adoc

```diff
@@ -187,24 +250,72 @@ Since version 1.1.0 of the Secure Configuration Properties module, you can encry
 </secure-properties:config>
 ----
 
-To encrypt a file content, see <<secure_props_tool>>.
+In Studio, in the Secure Configuration Properties window, select *True* for the *File level encryption*:
 
-[[qanda]]
-=== Frequently Asked Questions
+image::secure-configuration-properties-studio3.png[config extension3]
 
-*What if a property is defined in multiple files?*
 
-In that case, the actual property's value will be the one in which is first defined.
 
-*What if I want to define a secure property to depend on a previously defined secure property?*
 
-It is possible to define a property to depend on a previously defined one, just using the same syntax: `property=My dependent ${secure::dependent.property} value`.
+== Usage of Secure Configuration Properties
 
-Note that Secure Configuration Properties can depend on other Secure Configuration Properties, but not on other Mule configuration properties (see xref:configuring-properties.adoc[Configure Properties]).
+The following example shows how a flow uses the encrypted `encrypted.value1` value to set the payload. At runtime, the decryption algorithm is used to store the decrypted value of `encrypted.value1` into memory.
 
-== Warning
+*Use of Encrypted Secure Properties*
+[source,xml,linenums]
+----
+<flow name="main">
+    <set-payload value="${secure::encrypted.value1}"/>
+</flow>
+----
+
+The following example shows how flow uses a non-encrypted value from the same secure properties file:
+
+*Use of Non-encrypted Properties*
+[source,xml,linenums]
+----
+<flow name="mainNonEncrypted">
+    <set-payload value="${secure::testPropertyA}"/>
+</flow>
+----
+
+=== Set Secure Configuration Properties File Dynamically
+
+A common configuration use case is to set the file to depend on a property (for example, `env`) to determine which file to use. For example, to use a `development-properties` file in development stage (which could not be encrypted) or a production file (with encrypted values).
+
+*Secure Configuration Properties with Environment File Defined*
```

Comment:
ditto GLOBAL re. titles for graphics, examples, tables, lists, and so on.
Use intro text that ends with a colon instead.
I'll stop marking.